yt_dlp/extractor/franceculture.py

```diff
@@ -1,26 +1,35 @@
 # coding: utf-8
 from __future__ import unicode_literals
 
+import re
 from .common import InfoExtractor
 from ..utils import (
     determine_ext,
     extract_attributes,
     int_or_none,
+    traverse_obj,
 )
 
 
 class FranceCultureIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?franceculture\.fr/emissions/(?:[^/]+/)*(?P<id>[^/?#&]+)'
     _TESTS = [{
```

Comment:
Add all the keys u are returning to the tests, for example...the description

Reply:
What do you mean?

Reply:
add description and duration to the video test

Reply:
Done, thanks.